tests/e2e/gc_metadata_syncer.go

```diff
@@ -38,7 +38,7 @@ import (
 	admissionapi "k8s.io/pod-security-admission/api"
 )
 
-var _ = ginkgo.Describe("[csi-guest] pvCSI metadata syncer tests", func() {
+var _ = ginkgo.Describe("[csi-guest] [ef-vks] pvCSI metadata syncer tests", func() {
```

Comment:
If this is negetive test like service restart, Remove from EF

Reply:
done